### ISSUE TITLE
AVT2 - Content Switcher Component does not meet color contrast requirements 

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#1715]
Carbon Accessibility Sandbox (Carbon X) - [Content Switcher Component ](https://carbon-accessibility-sandbox.netlify.com/components/content-switcher/)

Tested on macOS Mojave 10.14.2, Chrome Version 71.0.3578.98 (Official Build) (64-bit), Color Contrast Analyzer

When the mouse hovers over an inactive tab the Color contrast ration between the light grey tabs and the hovered grey tab does not meet the requirement.
Color contrast ratio is 1.1:1 and it needs to be 3:1 to meet WCAG 2.1 (see screenshot)
<img width="480" alt="screen shot 2019-01-30 at 1 49 17 pm" src="https://user-images.githubusercontent.com/21676914/52008299-f9bab280-2495-11e9-9de2-10de9fdb4232.png">

**Comment by dakahn** [comment]
@carbon-design-system/design 👋
